Guake won't start on NixOS 19.03

## Issue description
I am trying to start the guake terminal under a gnome3 desktop and get 
```
[ERROR] Unable to start Guake, missing mandatory dependency: Keybinder 3
```
Looking at its derivation `keybinder3` is part of the `buildInputs` and should be there. Any ideas what's going wrong here? 

### Steps to reproduce
```
nix-env -i guake
guake
```

## Technical details
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.35, NixOS, 19.03.172286.8ea36d73256 (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2.2`
 - channels(root): `"nixos-19.03.172286.8ea36d73256"`
 - channels(markus): `"nixpkgs-19.03.172286.8ea36d73256"`

--- a/pkgs/applications/misc/guake/default.nix
+++ b/pkgs/applications/misc/guake/default.nix
@@ -14,6 +14,11 @@ in python3.pkgs.buildPythonApplication rec {
     sha256 = "1j38z968ha8ij6wrgbwvr8ad930nvhybm9g7pf4s4zv6d3vln0vm";
   };
 
+  # Strict deps breaks guake
+  # See https://github.com/NixOS/nixpkgs/issues/59930
+  # and https://github.com/NixOS/nixpkgs/issues/56943
+  strictDeps = false;
+
   nativeBuildInputs = [ gettext gobject-introspection wrapGAppsHook python3.pkgs.pip glibcLocales ];
 
   buildInputs = [ gtk3 keybinder3 libnotify python3 vte ];

